lms/djangoapps/program_enrollments/api/v1/tests/factories.py

```diff
@@ -20,3 +22,15 @@ class Meta(object):
     program_uuid = uuid4()
     curriculum_uuid = uuid4()
     status = 'enrolled'
+
+class ProgramCourseEnrollmentFactory(factory.DjangoModelFactory):
+    """
+    Factory for ProgramCourseEnrollment models
+    """
+    class Meta(object):
+        model = models.ProgramCourseEnrollment
+
+    program_enrollment = factory.SubFactory(ProgramEnrollmentFactory)
+    course_enrollment = factory.SubFactory(CourseEnrollmentFactory)
+    course_key = CourseKey.from_string("course-v1:edX+DemoX+Demo_Course")
+    status = "active"
```

Comment:
#NIT new line please

Reply:
yeah, i'm working my way through the quality issue as well